### PR TITLE
Friendly Gemfile: Remove Deprication Warnings and Add Licence File

### PR DESCRIPTION
**Pull request by heymatthew**
As a developer including the `harverstine_distance` gem in my project
I want no ruby warnings and an easy to find LICENCE file
So that I can quickly assess legal and compatibility requirements for my application

I kinda mashed these together, but wouldn't mind dropping the LICENCE if it doesn't feel right. Hope that's cool.

### CODE REVIEW

File: lib/haversine.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@ def toRad
   end
 end
 
-class Fixnum
+class Integer
```
Comment (Jirapong) [inline lib/haversine.rb]: It looks like this is true for Ruby 2.4 but not 2.3 . Am i correct?

Comment (heymatthew) [reply lib/haversine.rb]: Yes, that looks to be correct, the deprivation warnings started after 2.4.0.

I've run tests against ruby 2.4.0 and 2.3.6 and it passes on both :).

Comment (Jirapong) [reply lib/haversine.rb]: perfect. Thank you.

Comment (heymatthew) [reply lib/haversine.rb:15]: Legend 👍, gimme a shout if this turns out to need further polish.

Oh, and happy new year!

**Pull request merged**